ui/component/or-dashboard-builder/src/widgets/or-image-widget.ts

```diff
@@ -0,0 +1,366 @@
+import manager, {DefaultColor5, DefaultColor4, DefaultColor2, DefaultColor3, Util } from "@openremote/core";
+import { Asset, Attribute, AttributeRef, DashboardWidget } from "@openremote/model";
+import { showSnackbar } from "@openremote/or-mwc-components/or-mwc-snackbar";
+import { i18next } from "@openremote/or-translate";
+import { css, html, LitElement, TemplateResult, unsafeCSS } from "lit";
+import { customElement, property, state, query } from "lit/decorators.js";
+import { OrWidgetConfig, OrWidgetEntity } from "./or-base-widget";
+import { style } from "../style";
+import {debounce} from "lodash";
+import { SettingsPanelType, widgetSettingsStyling } from "../or-dashboard-settingspanel";
+import { InputType, OrInputChangedEvent } from "@openremote/or-mwc-components/or-mwc-input";
+
+export interface ImageWidgetConfig extends OrWidgetConfig {
+    displayName: string;
+    attributeRefs: AttributeRef[];
+    xCoordinatesMap: [number];
+    yCoordinatesMap: [number];
+    showTimestampControls: boolean;
+    imageUploaded: boolean;
+    imagePath: string;
+    assets: Asset[];
```

Comment:
`ImageWidgetConfig` is an object that also gets stored in the database.
We regularly use or `assetId` or `attributeRef` (which is assetId+attributeName), 
to prevent all asset details being stored in the database.

You can probably replace `assets` with `assetIds` in this case.